wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -179,3 +196,6 @@ sendMoney SendActions{..} passphrase moneySource dstDistr = do
      -- TODO eliminate copy-paste
      listF separator formatter =
          F.later $ fold . intersperse separator . fmap (F.bprint formatter)
+     getCachedUtxo = do
+         modifier <- getUtxoModifier
+         MM.modifyMap modifier <$> getWalletUtxo
```

Comment:
So the modifier here cuts out entries from the utxo that are 0. So then filtering with respect to this gives us just addresses with non-zero balance.